CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   to fetch vulnerability updates from Central. The flag will be removed as part of the 4.2.0 release.
 - ROX-15925: The KernelModule collection method is deprecated in favor of EBPF. This method will be removed in the 4.1 release.
 - Deprecated v1.0 of Network Graph. Please switch to the new 2.0 version for improved functionality and a better user experience.
+- The API endpoint `/v1/serviceaccounts` is deprecated and will be removed as part of the 4.2.0 release.
```

Comment:
My original wording was to deprecate the behavior of the endpoint so that we could retain the endpoint name more specifically. I don't think we necessarily need to change the endpoint name. Maybe we say the endpoint `...is deprecated and will be changed as part...`?

Reply:
`ListServiceAccounts` isn't very well known IMO. So perhaps in that case `/v1/serviceaccounts is deprecated and will be changed as part` does make more sense